src/toil_lib/urls.py

```diff
@@ -23,14 +23,15 @@ def download_url(job, url, work_dir='.', name=None, s3_key_path=None, cghub_key_
     :param str name: Name of output file, if None, basename of URL is used
     :param str s3_key_path: Path to 32-byte encryption key if url points to S3 file that uses SSE-C
     :param str cghub_key_path: Path to cghub key used to download from CGHub.
+    :param bool mock: Whether to run docker calls with mock_mode. See :func:`toil_lib.programs.docker_call`
```

Comment:
I was just asking for a minor spelling fix. `with mock_mode` is a bit awkward. I thought `in mock mode` sounded better. The semantics of this parameter is the same as for docker_call, correct? True enables mock mode, False disables it and None (the default) uses the environment variable or, if that's not set, ultimately disables it. Right?